plugins/Toolbox/resources/qml/ToolboxDetailTileActions.qml

```diff
@@ -66,6 +67,27 @@ Column
         }
     }
 
+    Label
+    {
+        property var whereToBuyUrl:
+        {
+            var pg_name = "whereToBuy"
+            return (pg_name in packageData.links) ? packageData.links[pg_name] : undefined
```

Comment:
This doesn't yet use the new 'type' field that was added to the API spec?

Reply:
![image](https://user-images.githubusercontent.com/1134120/52712869-2f7f9100-2f96-11e9-8e5b-9c190a7559fd.png)

Reply:
Hey @ChrisTerBeke , do you know what the `type`s will be for the following items (some existing `title`s in production):
 - technicalDataSheet
 - safetyDataSheet
 - printingGuidelines

I guess that they will be all lowercases with underscores?

Reply:
![image](https://user-images.githubusercontent.com/1134120/52787254-8e5c0d80-305d-11e9-8ace-528314e671d4.png)

Excerpt from https://api-staging.ultimaker.com/cura-packages/v1/spec.

Reply:
I'll keep it for now to use `title` because the change is not in production yet. I'll put a reminder for changing to use `type`.